### ISSUE TITLE
Training Advisor

### ISSUE THREAD

**Issue by Cosby5** [rmcc3/ai-toolkit-revamped#65]
## This is for bugs only

Did you already ask [in the discord](https://discord.gg/VXmU2f5WEU)?

Yes/No - No because this is directly about the fork and not the main project.

You verified that this is a bug and not a feature request or question by asking [in the discord](https://discord.gg/VXmU2f5WEU)?

Yes/No - It's a bug and a feature request.

## Describe the bug
If you are like me, you have one dataset folder with captions for each model type of model. For example, I have a `txt` with is a general description, `json` for a simpler JSON description (for ZIT and Flux.2, general datasets), `caption` for an extremely detailed JSON description (for the subject datasets), and `sdxl` for the models using similar schemas like Pony, Illustrious, ect. The issue is your tool flags these as my "Caption extension may be wrong". First it should have a toggle to turn that warning off, or at least some means to ignore things. 

The second issue is, when you set the optimizer to `Prodigy` you get a warning "Learning rate looks too aggressive" when setting the rate to the one requested in the docs. 

Long story short, maybe add something to toggle on and off the advisor. It's helpful but sometimes it's flagging intentional things, so it is just visual clutter.
